### PR TITLE
[improvement](config)Add the ShellCheck check-free project about Dockerfile

### PR DESCRIPTION
**Pull request by FreeOnePlus**
# Proposed changes

Add the ShellCheck check-free project about Dockerfile
^docker/runtime/fe/Dockerfile ^docker/runtime/be/Dockerfile

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [ ] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [ ] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by morningman — DISMISSED** [review]
LGTM

File: .github/workflows/shellcheck.yml [hunk 1]
```diff
@@ -49,4 +49,4 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           sh_checker_comment: true
-          sh_checker_exclude: .git .github ^thirdparty/src ^thirdparty/installed ^ui ^docs/node_modules ^tools/clickbench-tools ^extension ^output ^fs_brokers/apache_hdfs_broker/output
+          sh_checker_exclude: .git .github ^thirdparty/src ^thirdparty/installed ^ui ^docs/node_modules ^tools/clickbench-tools ^extension ^output ^fs_brokers/apache_hdfs_broker/output  ^docker/runtime/fe/Dockerfile ^docker/runtime/be/Dockerfile
```
Comment (adonis0147) [inline .github/workflows/shellcheck.yml:52]:
```suggestion
          sh_checker_exclude: .git .github ^thirdparty/src ^thirdparty/installed ^ui ^docs/node_modules ^tools/clickbench-tools ^extension ^output ^fs_brokers/apache_hdfs_broker/output (^|.*/)Dockerfile$
```

**Review by adonis0147 — CHANGES_REQUESTED** [review]
We should ignore all `Dockerfile`.

File: .github/workflows/shellcheck.yml (hunk 1 above)
Comment (FreeOnePlus) [reply .github/workflows/shellcheck.yml:52]: thks

**Review by adonis0147 — APPROVED** [review]
LGTM

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**